src/ai-store/browser.js

```diff
@@ -40,9 +40,9 @@ export async function getBrowserServiceData() {
 async function getBrowserAiCapabilities( ai ) {
 	const capabilities = [];
 
-	if ( ai.canCreateTextSession ) {
-		const supportsTextGeneration = await ai.canCreateTextSession();
-		if ( supportsTextGeneration && supportsTextGeneration === 'readily' ) {
+	if ( ai.assistant ) {
+		const supportsTextGeneration = await ai.assistant.capabilities();
+		if ( supportsTextGeneration && supportsTextGeneration.available === 'readily' ) {
```

Comment:
Prettier complains about this:
```suggestion
		if (
			supportsTextGeneration &&
			supportsTextGeneration.available === 'readily'
		) {
```

You could get this auto-fixed by running `npm run format-js`.